cruise.umple/src/class/UmpleInternalParser_CodeClass.ump

```diff
@@ -523,6 +529,9 @@ class UmpleInternalParser
     {
       analyzeException(token,aClass);
     }
+    else if (token.is("checkForUnintendedBracket")){
```

Comment:
This check should only be at the top level ... and hence should only be there once ...